src/chrome/content/rules/Ubisoft-Entertainment.xml

```diff
@@ -53,79 +28,67 @@ Fetch error: http://static16.ubi.com/ => https://s3.amazonaws.com/static16.ubi.c
 
 	ToDo: Secure shop pages that don't redirect to http.
```

Comment:
This comment is obsolete, `store.ubi.com` can be added as target.

Reply:
Fixed.